reporter.go

```diff
@@ -38,19 +40,19 @@ func registerStandardLambdaMetrics() {
 	invocationsCounterName := wavefront.DeltaCounterName(getStandardLambdaMetricName("invocations"))
 	wavefront.RegisterMetric(invocationsCounterName, invocationsCounter, nil)
 
-	// Register Error counter
+	// Register Error counter.
 	errCounter = metrics.NewCounter()
 	errCounterName := wavefront.DeltaCounterName(getStandardLambdaMetricName("errors"))
 	wavefront.RegisterMetric(errCounterName, errCounter, nil)
 
-	// Register duration gauge
+	// Register duration gauge.
 	durationGauge = metrics.NewGaugeFloat64()
 	wavefront.RegisterMetric(getStandardLambdaMetricName("duration"), durationGauge, nil)
 }
 
-// Method to send all metrics in the registry to wavefront.
+// reportMetrics sends the collected metrics in the registry to Wavefront. Together with the metrics, the
+// default point tags are also sent.
```

Comment:
> There are a few reporter level point tags added by default which are added to all metrics reported by the reporter.

Reply:
Added a small blurb that mentions where to find the standard point tags